j4l13n/vendorPro
# Add tables to database

### what does this PR do?

configuration of database and setup environment for the application.

### Description of work to be completed

Have the following endpoints working:

- GET /api/vendors 
- GET /api/members

Have the following configuration working

- PostgreSQL must be working with sequelize package

### How should this be manual verified

- After cloning the application check README file for configuration and start using
```
> npm start
```
remember to migrate all table to database before starting the application
```
> sequelize db:migrate
```
